src/main/kotlin/uk/gov/justice/hmpps/architecture/model/AdjudicationsApi.kt

```diff
@@ -0,0 +1,40 @@
+package uk.gov.justice.hmpps.architecture.model
+
+import com.structurizr.model.Container
+import com.structurizr.model.Model
+import com.structurizr.model.SoftwareSystem
+import com.structurizr.view.ViewSet
+import uk.gov.justice.hmpps.architecture.HMPPSSoftwareSystem
+import uk.gov.justice.hmpps.architecture.annotations.Tags
+
+class AdjudicationsApi private constructor() {
+  companion object : HMPPSSoftwareSystem {
+    lateinit var system: SoftwareSystem
+    lateinit var api: Container
+
+    override fun defineModelEntities(model: Model) {
+      system = model.addSoftwareSystem(
+        "Adjudications API",
+        "Allows reporting and viewing of Adjudications"
```

Comment:
Is the API the system itself, or does it sit under a bigger context? (What is the problem space?)

Reply:
w.r.t. Adjudications, this is going to be the single source of truth. But at the moment it manages temporary data that is sent to NOMIS via the Prison API.